arel_toolkit.gemspec

```diff
@@ -24,9 +24,8 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
   spec.extensions    = ['ext/pg_result_init/extconf.rb']
-
-  spec.add_dependency 'arel', '~> 9.0.0'
```

Comment:
'arel' was moved to 'activerecord' in Rails 6

Reply:
So interestingly enough you've removed arel from the gemspec, didn't add it to the Appraisal and everything still works? 🤔 . Does that mean Arel is required by ActiveRecord in version 5 and 6?

Reply:
Your assumption is right, `arel` is in runtime dependencies of ActiveRecord
https://rubygems.org/gems/activerecord/versions/5.0.0.1